docs/source/app-dev/app-arch.rst

```diff
@@ -148,6 +148,24 @@ To use command deduplication, you should:
 
 For more details on command deduplication, see the :ref:`Ledger API Services <command-submission-service-deduplication>` documentation.
 
+
+.. _handling-participant-node-failover:
+
+Handling participant node failover
+**********************************
+
+Some DAML Ledgers support exposing multiple eventually consistent Ledger API endpoints hosted by separate participant nodes.
+
+To support switching to a different participant node at run time,
+your application should keep track of the last ledger offset received from the :ref:`transaction service <transaction-service>`.
+After switching to a new participant node, subscribe to the transaction stream using the last received offset.
+If you receive the OUT_OF_RANGE error, it means the new participant node hasn't caught up with the state on the previous participant yet
```

Comment:
I'd suggest adding a sentence referencing the gRPC status code explanation and explaining that we use the OUT_OF_RANGE error to indicate a state that is not reached, but might be reached.